src/views/Exam.vue

```diff
@@ -1,6 +1,16 @@
 <template>
-    <div>
-        <v-progress-circular v-if="userQuiz === null" />
+    <div align="center">
```

Comment:
Why has this code been copied and pasted between views instead of using a single component?